### ISSUE TITLE
v0.8.13 breaks unit tests with "TypeError"

### ISSUE THREAD

**Issue by wolfio** [angular/zone.js#837]
using:
- zone.js 0.8.13
- node 6.10.3
- angular 4.2.5
- rxjs 5.4.2
- karma 1.7.0
- typescript 2.4.1
- jasmine 2.6.0

When using zone.js v0.8.12 unit tests work fine. With v0.8.13 I get this exception on every component test:

```
TypeError: Attempted to assign to readonly property. in node_modules/zone.js/dist/long-stack-trace-zone.js (line 115)
	onScheduleTask@node_modules/zone.js/dist/long-stack-trace-zone.js:115:22
	scheduleTask@node_modules/zone.js/dist/zone.js:404:65
	onScheduleTask@node_modules/zone.js/dist/zone.js:300:41
	scheduleTask@node_modules/zone.js/dist/zone.js:404:65
	scheduleTask@node_modules/zone.js/dist/zone.js:235:55
	scheduleEventTask@node_modules/zone.js/dist/zone.js:261:37
	node_modules/zone.js/dist/zone.js:1561:46
	addEventListener
	addEventListener
	listen
	listen
	listenToElementOutputs
	createViewNodes
	callViewAction
	execComponentViewsAction
	createViewNodes
	createRootView
	callWithDebugContext
	debugCreateRootView
	create
	initComponent
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/proxy.js:79:45
	invoke@node_modules/zone.js/dist/zone.js:390:40
	onInvoke
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	run
	createComponent
	createComponent
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/proxy.js:79:45
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/jasmine-patch.js:104:37
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/async-test.js:49:45
	onInvoke@node_modules/zone.js/dist/proxy.js:76:47
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/async-test.js:38:46
	invokeTask@node_modules/zone.js/dist/zone.js:424:36
	runTask@node_modules/zone.js/dist/zone.js:191:57
	invokeTask@node_modules/zone.js/dist/zone.js:499:41
	invoke@node_modules/zone.js/dist/zone.js:488:53
	timer@node_modules/zone.js/dist/zone.js:1734:34
	TypeError: Attempted to assign to readonly property. (line 59)
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/proxy.js:79:45
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/jasmine-patch.js:104:37
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/async-test.js:49:45
	onInvoke@node_modules/zone.js/dist/proxy.js:76:47
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/async-test.js:38:46
	invokeTask@node_modules/zone.js/dist/zone.js:424:36
	runTask@node_modules/zone.js/dist/zone.js:191:57
	invokeTask@node_modules/zone.js/dist/zone.js:499:41
	invoke@node_modules/zone.js/dist/zone.js:488:53
	timer@node_modules/zone.js/dist/zone.js:1734:34
	TypeError: undefined is not an object (evaluating 'element.querySelector') (line 65)
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/proxy.js:79:45
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/jasmine-patch.js:104:37
	invoke@node_modules/zone.js/dist/zone.js:391:31
	onInvoke@node_modules/zone.js/dist/async-test.js:49:45
	onInvoke@node_modules/zone.js/dist/proxy.js:76:47
	invoke@node_modules/zone.js/dist/zone.js:390:40
	run@node_modules/zone.js/dist/zone.js:141:49
	node_modules/zone.js/dist/async-test.js:38:46
	invokeTask@node_modules/zone.js/dist/zone.js:424:36
	runTask@node_modules/zone.js/dist/zone.js:191:57
	invokeTask@node_modules/zone.js/dist/zone.js:499:41
	invoke@node_modules/zone.js/dist/zone.js:488:53
	timer@node_modules/zone.js/dist/zone.js:1734:34
```
 
It seems this happens because the beforeEach within the component tests is not finished. It looks like it fails after calling TestBed.createComponent(MyComponent)

```
beforeEach(() => {
      fixture = TestBed.createComponent(MyComponent);
      component = fixture.componentInstance;
      element = fixture.debugElement.nativeElement;

      fixture.detectChanges();
    });
```

because code afterwards is not executed and therefore some objects are undefined.

**Comment by JiaLiPassion** [comment]
duplicate with #832, it have been fixed in #834.

**Comment by JiaLiPassion** [comment]
Close for now for fixed
